metis-enrichment/metis-enrichment-client/src/main/java/eu/europeana/enrichment/rest/client/dereference/DereferencerProvider.java

```diff
@@ -96,11 +97,17 @@ public Dereferencer create() throws DereferenceException {
       entityClientConfiguration = null;
     }
 
+    final Dereferencer dereferencer;
+    try {
```

Comment:
Hmm. Should this try be inside the constructor? Our class now throws EntityClientException which feels non natural if I may say. Perhaps it should through the DereferenceException from its constructor.